src/chronos/chronos.py

```diff
@@ -59,27 +59,19 @@ class ChronosTokenizer:
     which concrete classes must implement.
     """
 
-    def input_transform(
+    def context_input_transform(
         self,
         context: torch.Tensor,
-        tokenizer_state: Any = None,
-    ) -> Tuple[torch.Tensor, torch.Tensor, Any]:
+    ) -> Tuple:
         """
-        Turn a batch of time series into token IDs, attention map, and scale.
+        Turn a batch of time series into token IDs, attention map, and tokenizer_state.
```

Comment:
The docstring of the third return argument can be improved (can't comment on it directly). Instead of

> An object that will be passed to ``output_transform``

better

> An object that can be passed to ``label_input_transform`` and ``output_transform``

Reply:
Fixed.